lms/envs/devstack.py

```diff
@@ -129,8 +129,12 @@ def should_show_debug_toolbar(_):
 
 ########################## SECURITY #######################
 FEATURES['ENFORCE_PASSWORD_POLICY'] = False
-FEATURES['ENABLE_MAX_FAILED_LOGIN_ATTEMPTS'] = False
-FEATURES['SQUELCH_PII_IN_LOGS'] = False
+
+#FEATURES['ENABLE_MAX_FAILED_LOGIN_ATTEMPTS'] = False
+FEATURES['ENABLE_MAX_FAILED_LOGIN_ATTEMPTS'] = True #cl debug improve_account_lockout_messaging
```

Comment:
You could place this in your `lms/envs/private.py` file to avoid temporarily checking it in. Either way, you'll want to remove this before merging.